src/java/io/bazel/rulesscala/scalac/StreamReporter.java

```diff
@@ -0,0 +1,24 @@
+package io.bazel.rulesscala.scalac;
+
+import java.io.BufferedReader;
+import java.io.ByteArrayInputStream;
+import java.io.InputStreamReader;
+import java.io.PrintStream;
+import java.io.PrintWriter;
+import scala.tools.nsc.Settings;
+import scala.tools.nsc.reporters.ConsoleReporter;
+
+class StreamReporter extends ConsoleReporter {
```

Comment:
isn't this the exact same code as the `211` code? Can't we share or at least document why it is the same or how it is subtly different? (I understand they will be compiled with different versions of scala, but that's always the case when we change versions).

Reply:
The signature of the constructor is different for [2.11](https://github.com/scala/scala/blob/2e2f65a201b0b06e01a39fec4ae232c5b94efcca/src/compiler/scala/tools/nsc/reporters/ConsoleReporter.scala#L17) and for the ve[rsions after that](https://github.com/scala/scala/blob/ce02613de571004cf7d0e4cc22ae919d825c947f/src/compiler/scala/tools/nsc/reporters/ConsoleReporter.scala#L21)